WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/PrefMasterSwitchToolbarView.kt

```diff
@@ -230,8 +231,10 @@ class PrefMasterSwitchToolbarView @JvmOverloads constructor(
         } ?: if (BuildConfig.DEBUG) {
             throw IllegalStateException("Unknown view style id: $viewStyleInt")
         } else {
-            AppLog.e(AppLog.T.SETTINGS, "PrefMasterSwitchToolbarView.setViewStyle called from xml " +
-                    "with an unknown viewStyle.")
+            AppLog.e(
+                    AppLog.T.SETTINGS, "PrefMasterSwitchToolbarView.setViewStyle called from xml " +
+                    "with an unknown viewStyle."
+            )
```

Comment:
Any reason why these strings are separated? Could we change it to something like this:

```
AppLog.e(
        AppLog.T.SETTINGS,
        "PrefMasterSwitchToolbarView.setViewStyle called from xml with an unknown viewStyle."
)
```